### PR TITLE
add note to use the constraint UserPassword

### PR DESCRIPTION
**Pull request by zairigimad**
add note to use the constraint UserPassword

### CODE REVIEW

File: reference/constraints/UserPassword.rst [hunk 1]
```diff
@@ -10,6 +10,11 @@ password, but needs to enter their old password for security.
     This should **not** be used to validate a login form, since this is
     done automatically by the security system.
 
+.. note::
+
+    In order to use this constraints, you should have installed the 
+    symfony/security component with Composer.
```
Comment (HeahDude) [inline reference/constraints/UserPassword.rst]: ``symfony/security-core``

**Review by HeahDude — COMMENTED** [review]
Thanks, a minor comment

File: reference/constraints/UserPassword.rst (hunk 1 above)
Comment (zairigimad) [reply reference/constraints/UserPassword.rst]: Done :)

**Comment by OskarStark** [comment]
Thank you Zairig.

**Pull request merged**